src/plugins/saved_search/public/services/saved_searches/types.ts

```diff
@@ -18,3 +23,26 @@ export interface SavedSearch extends SavedSearchCommon {
     errorJSON?: string;
   };
 }
+
+interface SearchBaseInput extends EmbeddableInput {
+  timeRange: TimeRange;
+  timeslice?: [number, number];
+  query?: Query;
+  filters?: Filter[];
+  hidePanelTitles?: boolean;
+  columns?: string[];
+  sort?: SortOrder[];
+  rowHeight?: number;
+  rowsPerPage?: number;
+}
+
+export type SavedSearchByValueAttributes = Omit<SavedSearchAttributes, 'description'> & {
+  description?: string;
```

Comment:
`description` must be optional to conform to the expected by value input interface.